src/core/thread/csl_tx_scheduler.cpp

```diff
@@ -224,7 +224,7 @@ Mac::TxFrame *CslTxScheduler::HandleFrameRequest(Mac::TxFrames &aTxFrames)
     frame->SetTxDelay(txDelay);
     frame->SetTxDelayBaseTime(
         static_cast<uint32_t>(mCslTxNeighbor->GetLastRxTimestamp())); // Only LSB part of the time is required.
-    frame->SetCsmaCaEnabled(false);
+    frame->SetCsmaCaEnabled(true);
```

Comment:
Do we know why it was set to `false` before?

This is used with `TxDelay` and `TxDelayBaseTime`, which specify the desired transmission time. It may be helpful to check the impact of this change on different radio platform implementations.

I am concerned it may have unintended consequences.

Reply:
Yes, previously if `mTxDelay` is set, on nordic platform, it directly calls `nrf_802154_transmit_raw_at()` which keeps CCA enabled. We don't have the CCA behavior documented in our header file yet.